middleware.js

```diff
@@ -198,12 +197,14 @@ module.exports = function(options) {
 
       mkdirp(dirname(cssPath), '0700', function(err) {
         if (err) {
-          return error(err);
+          error(err);
+          cssDone = true;
+          return doneWriting();
```

Comment:
Makes sense.

>can you please illustrate what you mean?

For instance, some consumer might be reading the error and not existing as Sass middle ware is existing the pipeline for them.

With that being said, I don't have strong feelings about it. Consistent behavior wins. Thanks for elaboration. :)